mis_builder/tests/test_aep.py

```diff
@@ -363,8 +363,19 @@ def test_inactive_tax(self):
         self.aep.parse_expr(expr)
         self.aep.done_parsing()
 
+        tax_group = self.env["account.tax.group"].create(dict(name="test tax group"))
+
         tax = self.env["account.tax"].create(
-            dict(name="test tax", active=True, amount=0, company_id=self.company.id)
+            dict(
+                name="test tax",
+                active=True,
+                amount=0,
+                company_id=self.company.id,
+                tax_group_id=tax_group.id,
+                country_id=self.env["res.country"]
+                .search([("code", "=", "US")], limit=1)
+                .id,
```

Comment:
Should this not be the country of `self.company`?

Reply:
Company didnt have country set. Added it now.